tests/dla/test_lie_closure.py

```diff
@@ -268,3 +271,143 @@ def test_is_independent(self, ops, op, is_independent_true, tol):
         assert v1._pw_to_idx == vcopy._pw_to_idx
         assert v1._rank == vcopy._rank
         assert v1._num_pw == vcopy._num_pw
+
+
+dla11 = [
+    PauliSentence({PauliWord({0: "X", 1: "X"}): 1.0, PauliWord({0: "Y", 1: "Y"}): 1.0}),
+    PauliSentence(
+        {
+            PauliWord({0: "Z"}): 1.0,
+        }
+    ),
+    PauliSentence(
+        {
+            PauliWord({1: "Z"}): 1.0,
+        }
+    ),
+    PauliSentence({PauliWord({0: "Y", 1: "X"}): -1.0, PauliWord({0: "X", 1: "Y"}): 1.0}),
+]
+
+
+class TestLieClosure:
```

Comment:
I think we should have a test for `max_iterations`, in particular one where this criterion leaves the returned generator set incomplete, i.e. not algebraically closed.

Reply:
✅